clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -918,11 +992,41 @@ def _get_gcs_filesystem(self):
             raise GravitinoRuntimeException(
                 "Service account key is not found in the options."
             )
-        return importlib.import_module("gcsfs").GCSFileSystem(
-            token=service_account_key_path
+        return (
+            TIME_WITHOUT_EXPIRATION,
+            importlib.import_module("gcsfs").GCSFileSystem(
+                token=service_account_key_path
+            ),
         )
 
-    def _get_s3_filesystem(self):
+    def _get_s3_filesystem(self, fileset_catalog: Catalog, identifier: NameIdentifier):
+        try:
+            fileset: GenericFileset = fileset_catalog.as_fileset_catalog().load_fileset(
+                NameIdentifier.of(identifier.namespace().level(2), identifier.name())
+            )
+            credentials = fileset.support_credentials().get_credentials()
+        except (NoSuchCredentialException, CatalogNotInUseException) as e:
+            logger.warning("Failed to get credentials from fileset: %s", e)
```

Comment:
How about throw exception here?

Reply:
Directly throw exception??? What if the fieldset has not set credentials for old version?

Reply:
The exception you catch is `(NoSuchCredentialException, CatalogNotInUseException)`

Reply:
Can you clearly describe the issue? If I'm not wrong, you mean I should only catch `NoSuchCredentialException` and should not catch `CatalogNotInUseException`?

Reply:
done